paymentsheet/src/test/java/com/stripe/android/link/LinkButtonScreenshotTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.stripe.android.link
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.link.ui.LinkButton
+import com.stripe.android.utils.screenshots.PaparazziRule
+import com.stripe.android.utils.screenshots.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+internal class LinkButtonScreenshotTest {
```

Comment:
This is kinda hacky. We don't have access to paparazziRule in the link module, so I added this in the payment sheet module 🙈